Solutions/AzureKubernets/Analytic Rules/AKS-clusterrolebinding.yaml

```diff
@@ -0,0 +1,38 @@
+id: c3ac03d5-dbf2-49ab-a2de-83396a20a5fb
+name: AKS find cluster-admin clusterrolebinding
+description: |
+  'detects: kubectl create clusterrolebinding my-svc-acct-admin --clusterrole=cluster-admin.
```

Comment:
It is not clear why this would be a detection; can you provide more details as to why this would be?  If so, please update the description to make this clearer as to how this is something specifically malicious.

Reply:
@shainw  - Changed to hunting query